### PR TITLE
Add first query to the backend api

### PR DESCRIPTION
**Pull request by matheus-srm**
The query is still not working. Need to figure out why it can't import `pymongo` (fixed).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,9 @@ scraper/include
 scraper/lib
 scraper/share
 scraper/pyvenv.cfg
-*/__pycache__
\ No newline at end of file
+backend/__pycache__
+backend/db/__pycache__
```
Comment (Costa-SM) [inline .gitignore]: There is no need for all these __pycache__ exclusions

File: backend/db/db.py [hunk 2]
```diff
@@ -14,9 +14,11 @@ def __init__(self):
...
-        return self._client["Sentzer"]
+        self._database = self._client["Sentzer"]
 
     def get_collection(self, collection_name: str):
-        database = self.get_database()
-        return database[collection_name]
+        return self._database[collection_name]
+
+
+def get_db():
```
Comment (Costa-SM) [inline backend/db/db.py:23]: Is this indented correctly ?

Comment (matheus-srm) [reply backend/db/db.py:23]: Yep

**Review by Costa-SM — COMMENTED** [review]
lgtm

**Review by Costa-SM — COMMENTED** [review]
lgtm

**Review by Costa-SM — APPROVED** [review]
lgtm. i'm stupid

**Pull request merged**